docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -372,7 +372,7 @@ Role sync allows you to map user roles from an identity provider to Grafana. To
 1. In the configuration file, set [`assertion_attribute_role`]({{< relref "../../../configure-grafana/enterprise-configuration#assertion_attribute_role" >}}) option to the attribute name where the role information will be extracted from.
 1. Set the [`role_values_editor`]({{< relref "../../../configure-grafana/enterprise-configuration#role_values_editor" >}}) option to the values mapped to the `Editor` role.
 1. Set the [`role_values_admin`]({{< relref "../../../configure-grafana/enterprise-configuration#role_values_admin" >}}) option to the values mapped to the organization `Admin` role.
-1. Set the [`role_values_grafana_admin`]({{< relref "../../../configure-grafana/enterprise-configuration#role_values_grafana_admin" >}}) option to the values mapped to the `Grafana Admin` role.
+1. Set the [`role_values_grafana_admin`]({{< relref "../../../configure-grafana/enterprise-configuration#role_values_grafana_admin" >}}) option to the values mapped to the `Grafana Admin` role. You need to quote the role value in role_values_grafana_admin
```

Comment:
```suggestion
1. Set the [`role_values_grafana_admin`]({{< relref "../../../configure-grafana/enterprise-configuration#role_values_grafana_admin" >}}) option to the values mapped to the `Grafana Admin` role. You must add double quotes around the role value, for example `"superadmin"`.
```

Reply:
I've looked at the code, and you only need to enclose the value in quotes if it contains a comma or a space. So `role_values_grafana_admin = superadmin` would actually work fine. However, `role_values_grafana_admin = my superadmin` would translate to two different roles - `my` and `superadmin`. So if you want to map a role that contains a space or a comma, you need to wrap this role in double quotes, eg `role_values_grafana_admin = "my superadmin"`.

The same parsing is used for `allowed_organizations`, `org_mapping`, `role_values_editor` and `role_values_admin`.

I wonder if it would be better to call this out in [the table here](https://github.com/grafana/grafana/blob/v9.5.2/docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md#configure-saml-authentication-in-grafana-1). What do you think?

We could also extend the examples [here](https://github.com/grafana/grafana/blob/v9.5.2/docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md#configure-role-sync) and update the info [here](https://github.com/grafana/grafana/blob/v9.5.2/docs/sources/setup-grafana/configure-grafana/enterprise-configuration/index.md#authsaml).